app/views/guests/show.html.erb

```diff
@@ -0,0 +1,14 @@
+<p> Name: <%= @guest.name %> </p>
+<% @guest.rooms.each do |room| %>
+  <p> Hotel: <%= room.hotel.name %> </p>
+  <p> Room: <%= room.suite %> </p>
+  <p> Rate: <%= room.rate %> </p><br>
+<% end %>
+
+<%= "Add a Room to this Guest" %>
+<%= form_with url: "/guest_rooms/#{@guest.id}", method: :post, local: true do |form| %>
```

Comment:
Talked about this already in your `routes.rb` file but the path of this form should be: `"/guests/#{@guest.id}/rooms"`.